src/SOS/Strike/strike.cpp

```diff
@@ -1231,6 +1237,7 @@ DECLARE_API(DumpMT)
     table.WriteRow("ComponentSize:", PrefixHex(vMethTable.ComponentSize));
     table.WriteRow("DynamicStatics:", vMethTable.bIsDynamic ? "true" : "false");
     table.WriteRow("ContainsPointers:", vMethTable.bContainsPointers ? "true" : "false");
+    // FIXME: "Slots in VTable" is a misnomer. This is the number of slots in the MethodTable
```

Comment:
Putting a FIXME here will just sit around for years.  You could just change the output here.

I don't have a strong opinion on the text (vtable is clear enough for me) but if you want to change it to something else, go for it (as long as you update the tests).  It won't hurt anything.